README.md

```diff
@@ -1,131 +1,90 @@
 
-# ckanext-followy
-A CKAN extension to show the datasets a user is following.
+Followy CKAN Extension
+=========================
 
-------------
-Requirements
-------------
+A CKAN extension that displays the datasets a user is following on their profile and dashboard in the `Following` tab.
 
-For example, you might want to mention here which versions of CKAN this
-extension works with.
 
-
-------------
-Installation
+Requirements
```

Comment:
Would it be a good idea to remove this section if the extension does not require any requirements of it's own